Autopilot/Inc/SensorFusion.hpp

```diff
@@ -30,6 +28,58 @@ struct SFOutput_t {
     float longitudeSpeed; //m/s
 };
 
+#ifndef ATTITUDE_DATATYPES_HPP
+//Following structs store the raw sensor data so other modules can have direct access to them without including sensor header files
+struct IMU_Data_t
+{
+    float magx, magy, magz;
+    float accx, accy, accz;
+    float gyrx, gyry, gyrz; 
+
+    bool isDataNew; 
+    int sensorStatus; 
+    float utcTime; 
+};
+
+struct Airspeed_Data_t
+{
+    double airspeed;        
+
+    int sensorStatus;       
+    bool isDataNew;         
+    float utcTime;          
+};
+#endif
```

Comment:
In the file `attitudeDatatypes.hpp`, delete the declarations of these structs. 

Then, you can remove the preprocessing statement (`#ifndef`) here